addon/components/discover-page/template.hbs

```diff
@@ -18,7 +18,7 @@
                         <h1> {{discoverHeader}} </h1>
                     {{/if}}
                     {{!POWERED BY SHARE}}
-                    <p class="lead">{{t 'eosf.components.discoverPage.poweredBy'}} <a href="https://share.osf.io/" id='share-logo' class="share-logo" title="SHARE" onclick={{action "click" "link" "Discover - SHARE Logo"}}></a></p>
+                    <p class="lead">{{t 'eosf.components.discoverPage.poweredBy'}} <a href="https://share.osf.io/" id='share-logo' style="background-image: url('./powered_by_share.png')" class="share-logo" title="SHARE" onclick={{action "click" "link" "Discover - SHARE Logo"}}></a></p>
```

Comment:
Actually, I don't think this is right. It should be using the assets url generating component / helper thing.